src/app/body/task-details/create-git-issue/create-git-issue.component.ts

```diff
@@ -54,7 +54,7 @@ export class CreateGitIssueComponent implements OnInit {
     this.gitToken=this.teamService.teamsDataJson[this.task.TeamId].GitToken;
     this.gitToken=atob(this.teamService.teamsDataJson[this.task.TeamId].GitToken);
     this.showClose = false;
-    this.teamId = this.taskId.slice(0,3);
+    this.teamId = this.teamService.teamsDataJson[this.task.TeamId].TeamId;
     if(this.startService.showTeams) {
```

Comment:
Its better to use `this.teamId = this.task.TeamId `
instead of getting teamId with TeamId